### PR TITLE
Feat: add Osano MGMT

### PR DESCRIPTION
**Pull request by flavioriper**
https://cmp.osano.com/16CPXbTOi1sXx4D3/1e6b223c-ed10-4c4b-a442-48fea69f76af/osano.js

### CODE REVIEW

**Review by jshearer — COMMENTED** [review]
I'm going to hold off on approving this until @dyaffe says go for it, since it also adds Hotjar again. We've added and removed hotjar in the past because it hurts our google pagespeed numbers when we add it, so I don't want to go back and forth again. If you throw up a PR that _just_ adds Osano I'll approve it now 👍

**Comment by dyaffe** [comment]
Why are we adding Hotjar?  As Joseph mentioned, I'm pretty anxious about adding that in due to page speed issues with Google SEO metrics.

**Comment by flavioriper** [comment]
Removed the hotjar until we figure out a better solution to it

File: gatsby-config.ts [hunk 1]
```diff
@@ -125,6 +125,12 @@ const cfg: GatsbyConfig = {
                 },
             },
         },
+        {
```
Comment (jshearer) [inline gatsby-config.ts:128]: Ok I heard back from @dyaffe: can we remove this `vwo` stuff for now please? Once that's done this is good to go

**Review by jshearer — CHANGES_REQUESTED** [review]

File: gatsby-config.ts (hunk 1 above)
Comment (flavioriper) [reply gatsby-config.ts:128]: vwo removed

File: gatsby-config.ts [hunk 2]
```diff
@@ -125,6 +125,12 @@ const cfg: GatsbyConfig = {
...
+            resolve: "gatsby-plugin-vwo-smartcode",
```
Comment (jshearer) [inline gatsby-config.ts:129]: This still needs to be removed, build will fail otherwise

**Review by jshearer — CHANGES_REQUESTED** [review]

**Review by jshearer — APPROVED** [review]
LGTM

**Pull request merged**